browser/js/content.js

```diff
@@ -288,7 +294,7 @@ function renderSuccess(el, closeCallback, extraHTML) {
 }
 
 // Render an error that replaces the body of the widget.
-function renderErrorFull(el, bodyHTML) {
+function renderErrorFull(el, closeCallback, bodyHTML) {
```

Comment:
I have a fairly big refactor going on in the next branch, v1.9, may clean this up there.